week04_approx_rl/homework_tf.ipynb

```diff
@@ -87,7 +147,7 @@
     "from gym.core import ObservationWrapper\n",
     "from gym.spaces import Box\n",
     "\n",
-    "import cv2\n",
+    "import skimage\n",
```

Comment:
Please do not suggest using `skimage.imresize()`, as it is roughly an order of magnitude slower than `cv2.resize()`.